### ISSUE TITLE
Bad error when requiring a file with an unsupported loader

### ISSUE THREAD

**Issue by weaverryan** [symfony/webpack-encore#56]
To reproduce, in any JavaScript file:

```js
require('./foo.json');
```

As there is no loader configured for `.json` files, you'll get an error:

> This relative module was not found:
> * ./foo.json in ./app/Resources/assets/js/app.js

This makes it appear that the file wasn't found. The bug is actually inside of friendly-errors-webpack-plugin: https://github.com/geowarin/friendly-errors-webpack-plugin/pull/32

**Comment by stof** [comment]
shouldn't we define the json loader by default ?

**Comment by weaverryan** [comment]
@stof I can't think of any reason not to :). But the issue will still happen for other file extensions.

**Comment by stof** [comment]
@weaverryan I totally agree with you about getting a better error message. But we should avoid to have this error happening for JSON.

**Comment by weaverryan** [comment]
I mixed this issue up with another. JSON works already (it just does in Webpack 2), and the *actual* bug in friendly errors is when you setup a rule/loader, and typo the rule name (e.g. `sassss-loader`). That's not really something that can happen in Encore. If you don't have `sass-loader` installed, we catch that before running the built.

So, closing - I think I saw this issue originally while putting Encore together.
